src/Chapter1.hs

```diff
@@ -557,7 +557,9 @@ True
 False
 -}
 isVowel c = error "isVowel: not implemented!"
-
+isVowel c
+  | c `elem` "aeiou" = True
```

Comment:
Absolutely correct! 👍🏼 

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎 

Also, since `elem` already returns `Bool`, you don't need to use guards. You can just do:

```haskell
isVowel c = c `elem` "aeiou"
```